### ISSUE TITLE
Change Docker images to proper version format

### ISSUE THREAD

**Issue by samjabrahams** [fluxcapacitor/utils.ml#2]
Version names shouldn't be built into the base name of the image, instead they should trail after a colon at the end of the name.

Example:

```
fluxcapacitor/utils-ubuntu-14.04
```

should become

```
fluxcapacitor/utils-ubuntu:14.04
```

It'll make versioning easier, and is the proper way to handle Docker naming.

**Comment by cfregly** [comment]
right, except that 14.04 is the version of Ubuntu.  we will have our own versions of the ubuntu-14.04 image as we add small things like curl and wget and stuff.

(I went thru the same thought process.)

remember, these are specific to us so it's ok if we choose our own convention.
